docs/user/advanced.rst

```diff
@@ -1097,7 +1097,7 @@ The **connect** timeout is the number of seconds Requests will wait for your
 client to establish a connection to a remote machine (corresponding to the
 `connect()`_) call on the socket. It's a good practice to set connect timeouts
 to slightly larger than a multiple of 3, which is the default `TCP packet
-retransmission window <https://www.hjp.at/doc/rfc/rfc2988.txt>`_.
+retransmission window <https://datatracker.ietf.org/doc/html/rfc2988>`_.
```

Comment:
> It's a good practice to set connect timeouts to slightly larger than a multiple of 3, which is the default [TCP packet retransmission window](https://www.hjp.at/doc/rfc/rfc2988.txt).

Despite the fact I already approved this PR, there's still a room for improvement.
[RFC 2988](https://datatracker.ietf.org/doc/html/rfc2988) was obsoleted by [RFC 6298](https://datatracker.ietf.org/doc/html/rfc6298), so this link might be updated even further, but it would require additional changes since newer RFC makes the whole statement not correct due to [this change](https://datatracker.ietf.org/doc/html/rfc6298#section-7).
So this might be handled in an additional PR.

Reply:
Thanks - seems like a sensible comment, agree it makes sense for an additional PR.